src/rulelib.cpp

```diff
@@ -595,59 +601,6 @@ ruleset_delete(rule_t *rules, int nrules, ruleset_t *rs, int ndx)
 	return;
 }
 
-/*
- * We create random rulesets for testing and for creating initial proposals
- * in MCMC
- */
-int
-create_random_ruleset(int size,
```

Comment:
Why did you remove this?

Reply:
Because the RANDOM_RANGE macro would cause compilation errors on Windows, and we weren't using this function anywhere.